Libraries/LibCore/GetPassword.cpp

```diff
@@ -48,13 +48,18 @@ Result<String, int> get_password(const StringView& prompt)
     char* password = nullptr;
     size_t n = 0;
 
-    int ret = getline(&password, &n, stdin);
+    auto line_length = getline(&password, &n, stdin);
     tcsetattr(STDIN_FILENO, TCSAFLUSH, &original);
     putchar('\n');
-    if (ret < 0) {
+    if (line_length < 0) {
         return errno;
     }
```

Comment:
I think this is a mistake: we are returning `errno` if ret/line_length is less than 0.

We probably mean to return `getline`'s `errno` here, not `tcsetattr`'s or `putchar`'s.

Is that the case?

Reply:
I'm going to assume that is the case, I'll fix that too.

Reply:
Oh, I noticed that we are returning the errno of the other `tcsetattr()`'s errno some lines before. I'm not sure what to do here since I don't know what the error value should be.

Reply:
But I don't think it makes sense to bail out if we cannot restore the terminal setting since we already got the password.